db/schema.rb

```diff
@@ -10,6 +10,52 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2019_05_20_135055) do
 
+  create_table "groups", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "notes", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.bigint "user_id"
+    t.string "title"
+    t.string "content"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_notes_on_user_id"
+  end
+
+  create_table "tasks", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.bigint "user_id"
+    t.string "title"
+    t.string "content"
+    t.datetime "due_date"
```

Comment:
nên để `not null` ko anh :v?

Reply:
a nghĩ k nên, nó vẫn có thể tạo đc task ko có due date mà nhỉ, nhưng vẫn có thể mark trạng thái là đã done hay inprogress,...

Reply:
task mà ko có due date thì là note r mà a :v

Reply:
> a nghĩ k nên, nó vẫn có thể tạo đc task ko có due date mà nhỉ, nhưng vẫn có thể mark trạng thái là đã done hay inprogress,...